vecfixed/src/lib.rs

```diff
@@ -0,0 +1,107 @@
+//noinspection ALL
+/// VecFixed is basically a vector that keep a fixed size. Every time new element is pushed
+/// to the vector, the oldest element is removed and the latest pushed is added to the end.
+#[derive(Default)]
+pub struct VecFixed<const N: usize, T: Default + ToString> {
+    start: usize,
```

Comment:
What do you think about changing `start` to something like `next_index`?

Reply:
Nice